packages/core/lib/report-from-error.js

```diff
@@ -12,6 +12,6 @@ module.exports = (maybeError, handledState) => {
     handledState,
     maybeError
   )
-  if (maybeError !== actualError) report.updateMetaData('error', 'non-error value', String(maybeError))
+  if (maybeError !== actualError) report.updateMetaData('error', 'non-error value', JSON.stringify(maybeError))
```

Comment:
Expected behaviour (in this specific case):
![image](https://user-images.githubusercontent.com/23224757/61773941-21ac2280-adf6-11e9-9a11-5c8a424d3cf0.png)